packages/graph/hash_graph/tests/integration/postgres/property_type.rs

```diff
@@ -1,25 +1,33 @@
 use crate::postgres::DatabaseTestWrapper;
+pub use crate::test_data::{data_type, property_type};
 
 #[test]
 fn insert() {
-    let age_pt = serde_json::from_str(crate::test_data::property_type::AGE_V1)
-        .expect("could not parse property type");
+    let age_pt =
+        serde_json::from_str(property_type::AGE_V1).expect("could not parse property type");
 
     let mut database = DatabaseTestWrapper::new();
+    database
+        .seed([data_type::NUMBER_V1], [])
+        .expect("Could not seed database");
+
     database
         .create_property_type(age_pt)
         .expect("could not create property type");
 }
 
 #[test]
 fn query() {
-    let interests_pt = serde_json::from_str(crate::test_data::property_type::INTERESTS_V1)
+    let favourite_quote_pt = serde_json::from_str(property_type::FAVOURITE_QUOTE_V1)
```

Comment:
favorite* 😞

Reply:
Ah damn. Once we had two versions of Favorite Quote. v1 was BE, v2 was AE, at one point I probably removed the wrong one 😄 
Fixed in 9b23a1470b597f48fd2b393684f92d3adcd76b25